src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -223,12 +228,12 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         log.debug("REST request to get ModelingExercise Statistics for Exercise: {}", exerciseId);
         var modelingExercise = modelingExerciseRepository.findByIdElseThrow(exerciseId);
         authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.TEACHING_ASSISTANT, modelingExercise, null);
-        if (compassService.isSupported(modelingExercise)) {
-            return ResponseEntity.ok(compassService.getStatistics(exerciseId).toString());
-        }
-        else {
-            return notFound();
-        }
+        // if (compassService.isSupported(modelingExercise)) {
+        // return ResponseEntity.ok(compassService.getStatistics(exerciseId).toString());
+        // }
+        // else {
+        return notFound();
+        // }
```

Comment:
```suggestion
        return notFound();
```

Reply:
This will be activated again in a future PR that is why it is commented out.